src/window.rs

```diff
@@ -682,6 +682,26 @@ impl Window {
     pub fn set_ime_position<P: Into<Position>>(&self, position: P) {
         self.window.set_ime_position(position.into())
     }
+
+    /// Requests user attention to the window, this has no effect if the application
+    /// is already focused. How requesting for user attention manifests is platform dependent,
+    /// see `UserAttentionType` for details.
+    ///
+    /// Providing `None` will unset the request for user attention.
+    ///
+    /// Unsetting the request for user attention might not be done automatically by the WM
+    /// when the window receives input.
```

Comment:
```suggestion
    /// Providing `None` will unset the request for user attention. Unsetting the request for user attention might not be done automatically by the WM
    /// when the window receives input.
```

I think these can be combined, just to have text that is a bit easier to just read, rather than a huge list of disconnected points.